exercises/patterns/printSolidSquare/myownsquare.js

```diff
@@ -0,0 +1,15 @@
+function printSolidSquare(height) {
+    newLine = "\r\n"
+    square =""
+    for (let i = 0 ; i < height; i ++){
+        for (let j = 0; j < height; j ++){
+            square = square + "#"}
```

Comment:
You can also do `console.log('#'.repeat(height))` to print `#` `height` many times.